src/Sulu/Component/Webspace/Webspace.php

```diff
@@ -495,10 +495,14 @@ public function getTemplates()
      *
      * @param string $type
      * @param string $template
+     * @param string|null $parentTemplate
      */
-    public function addDefaultTemplate($type, $template)
+    public function addDefaultTemplate($type, $template, $parentTemplate = null)
```

Comment:
In e.g. `addLocalization` we are passing the `Localization` object instead of its properties. Would handle it the same way here.

Reply:
The object is passed instead of its properties